controller_manager/CMakeLists.txt

```diff
@@ -141,7 +141,7 @@ if(BUILD_TESTING)
     test_chainable_controller
     ros2_control_test_assets::ros2_control_test_assets
   )
-  set_tests_properties(test_controller_manager_srvs PROPERTIES TIMEOUT 120)
+  set_tests_properties(test_controller_manager_srvs PROPERTIES TIMEOUT 240)
```

Comment:
That's 4 minutes... which non-simulation / integration tests raises some :triangular_flag_on_post: Do you have an idea how long this test takes in two conditions?

1) Ideal "decent" hardware"
2) a container limited to 1 or 2 CPUs

I believe the default free tier of GitHub action runners are 2 vCPUs...
However, if for example this was run on the highest performance GitHub hosted Linux runners with 64 vCPUs this 4 minutes would cost over 1$ which could add up quickly.